src/transformers/models/splinter/tokenization_splinter_fast.py

```diff
@@ -0,0 +1,136 @@
+# coding=utf-8
+# Copyright Ori Ram, Yuval Kirstain, Jonathan Berant, Amir Globerson, Omer Levy and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tokenization classes for Splinter."""
+from typing import List, Optional
+
+from ...utils import logging
+from ..bert.tokenization_bert_fast import BertTokenizerFast
+from .tokenization_splinter import SplinterTokenizer
+
+
+logger = logging.get_logger(__name__)
+
+VOCAB_FILES_NAMES = {"vocab_file": "vocab.txt"}
+
+PRETRAINED_VOCAB_FILES_MAP = {
+    "vocab_file": {
+        "splinter-base": "https://huggingface.co/splinter-base/resolve/main/vocab.txt",
+        "splinter-large": "https://huggingface.co/splinter-large/resolve/main/vocab.txt",
```

Comment:
These URLs need fixing as well.

Reply:
I wasn't sure what the URLs will be as the models haven't been uploaded yet. 
Any thoughts/suggestions? @sgugger

Reply:
If you do upload them to `tau/splinter-base` and `tau/splinter-large`, you will just need to add the organization in the URLs.